### ISSUE TITLE
Broken VENDOR_PREFIX detection on Minefield

### ISSUE THREAD

**Issue by charettes** [madrobby/scripty2#14]
It looks like firefox is dropping the style.MozBorderRadius in favor of style.borderRadius which breaks VENDOR_PREFIX detection AND VENDOR_MAP.

We should try to find another way of populating the VENDOR_MAP. Maybe by iterating through a list of style properties scripty2 depends on by testing the presence of the property and fallback appending possible prefixes (webkit Moz) and cache it.

I'll have a look at it tomorrow.

**Comment by savetheclocktower** [comment]
Yeah, the vendor prefix stuff was my way of prying open this gigantic can of worms. It definitely needs a more solid implementation. Thanks for taking a stab at it!

**Comment by charettes** [comment]
Let me know what you think of my approach @ http://github.com/charettes/scripty2/commits/fix-vendor-map.
